### PR TITLE
PD-498: Update MongoNav styles

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

Style updates to MongoNav based on the latest design spec

🎟 _Jira ticket:_ [PD-498](https://jira.mongodb.org/browse/PD-498)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/mongo-nav/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@leafygreen-ui/badge": "^3.0.0",
     "@leafygreen-ui/button": "^4.0.0",
     "@leafygreen-ui/hooks": "^2.0.0",
-    "@leafygreen-ui/icon-button": "^4.0.0"
+    "@leafygreen-ui/icon-button": "^4.0.0", 
+    "@leafygreen-ui/leafygreen-provider": "1.0.0"
```
Comment (DesignerDave) [inline packages/mongo-nav/package.json]: Let's be consistent about our versioning:
```
"@leafygreen-ui/leafygreen-provider": "^1.0.0"
```

File: packages/mongo-nav/src/MongoNav.tsx [hunk 2]
```diff
@@ -238,6 +241,7 @@ export default function MongoNav({
         onOrganizationChange={onOrganizationChange}
         admin={admin}
         hosts={sanitizedHosts}
+        currentProjectName={currentProject && currentProject.projectName}
```
Comment (DesignerDave) [inline packages/mongo-nav/src/MongoNav.tsx]: Optional chaining here?

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 3]
```diff
@@ -152,92 +260,79 @@ export default function OrgNav({
...
-            >
-              All Clusters
-            </a>
-          }
+        <LinkElement
+          href={orgNav.allClusters}
+          isActive={activeNav === 'allClusters'}
+          tooltipText="View all clusters across your organizations"
+          className={css`
+            margin-right: 25px;
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx]: Hmm... There's gotta be a better place to put this margin, right?

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 4]
```diff
@@ -152,92 +260,79 @@ export default function OrgNav({
...
         {admin && (
           <a
             href={orgNav.admin}
             className={cx(rightSideLinkStyle, linkText, {
               [activeLink]: activeNav === 'admin',
+              [navItemFocusStyle]: showFocus,
             })}
           >
-            Admin
+            <span>Admin</span>
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx]: Why the span?

File: packages/mongo-nav/package.json (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/package.json]: 👍

File: packages/mongo-nav/src/MongoNav.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/MongoNav.tsx]: 🤦‍♀ of course

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 4 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx]: I think I was originally styling it but then moved that elsewhere, my bad!

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 3 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx]: I thought if it was just one rule we were inlining now, but let me know if you want me to move

Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx]: nvm 👍

File: packages/mongo-nav/src/helpers/InteractionRingWrapper.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+const InteractionRingWrapper = ({
+  className,
+  ringClassName,
+  children,
+  selector,
+}: InteractionRingWrapperProps) => {
+  const { usingKeyboard: showFocus } = useUsingKeyboardContext();
+  const [hasFocus, setHasFocus] = useState(false);
+
+  const interactionRingFocusStyle = css`
```
Comment (hswolff) [inline packages/mongo-nav/src/helpers/InteractionRingWrapper.tsx]: Can we move these out of the component so they're only defined once?

**Comment by hswolff** [comment]
Rather, go ahead and merge w/o me.

**Comment by seanmhanson** [comment]
@DesignerDave ready for a quick pass. Prettier got upset at me, so there are two tiny spacing/comma things that seem to change and then change back between the two commits. Apologies, I can go back, squash, rebase, and replace those commits if you want, but we're close to merge so I don't want to delay things!

@brookescarlett everything was caused by three tiny things changed no doubt during merge:
- the data-testid prop for Org Triggers was removed
- the disabled prop for Org Triggers was removed
- the OrgNavLink component didn't spread rest params, so they each ignored their data-testid values in testing

**Pull request merged**